puppeteer.setup.js

```diff
@@ -1,6 +1,6 @@
 /* global page */
 
-const puppeteer = require('puppeteer');
+const puppeteer = require('puppeteer-core');
```

Comment:
add `targetEnv` to global list varaibles
![image](https://user-images.githubusercontent.com/63335823/195343677-aa3fa548-deca-4f72-975f-b1da9ccd13b9.png)